.env_sample

```diff
@@ -0,0 +1,3 @@
+GOOGLE_ANALYTICS_TRACKING_ID=GA1
```

Comment:
There shouldn't be any errors as the values will be defaulted to empty which should prevent the code branches where the includes are added from being executed. AFAIK the convention in the other repos is not to have any sample env files within the repo.
When specific things wish to be tested I'd probably look to either set the values on the command line or within the `docker-compose` files.